cli/rds.go

```diff
@@ -153,7 +153,7 @@ func createRdsImpl(client rdsClient, name string, username string, password stri
 		ConnectorJarURL:    jarURL,
 	}
 	if oracle != nil {
-		rdsRequest.OracleParameters = oracle
+		rdsRequest.Oracle = oracle
```

Comment:
For this you need to change the API, right?

Reply:
yep this is why it depends on hortonworks/cloudbreak#3453